aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -33,18 +52,19 @@ class DeleteModal extends Component {
   }
 
   render () {
-    const messages = {
-      header: this.props.header || 'Delete ',
-      objectType: this.props.objectType || 'pipeline'
-    }
+    const { formatMessage } = this.props.intl
     const header = (
       <span>
-        <FormattedMessage
-          id='delete.modal.header'
-          defaultMessage='{header} {objectType} '
-          values={messages}
+        <FormattedHTMLMessage
+          {...
+          { ...MESSAGES.delete,
+            values: {
+              obj: formatMessage(MESSAGES.objectType, { objType: this.props.objectType }),
```

Comment:
```javascript
 obj: formatMessage(MESSAGES[this.props.objectType])
```

Reply:
With this you get  an error `Uncaught Invariant Violation: [React Intl] An `id` must be provided to format a message.`

Reply:
let me check...

Reply:
![image](https://user-images.githubusercontent.com/8459537/58467751-84f23f00-813c-11e9-81d7-a4ca03d1749a.png)


```javascript

const MESSAGES = defineMessages({
  delete: {
    id: 'modal.delete.text',
    defaultMessage: 'Delete {obj} <b>{objName}</b>'
  },
  submissions: {
    id: 'modal.delete.submissions.text',
    defaultMessage: 'Data <b>submitted to</b> this pipeline (Submissions)'
  },
  entities: {
    id: 'modal.delete.entities.text',
    defaultMessage: 'Data <b>created by</b> this {obj} (Entities)'
  },
  pipeline: {
    id: 'modal.delete.object.type.pipeline',
    defaultMessage: 'pipeline'
  },
  contract: {
    id: 'modal.delete.object.type.contract',
    defaultMessage: 'contract'
  }
})

class DeleteModal extends Component {
  // ...

  render () {
    const { formatMessage } = this.props.intl
    const objType = formatMessage(MESSAGES[this.props.objectType || 'pipeline'])
    const header = (
      <span>
        <FormattedHTMLMessage {...
          { ...MESSAGES.delete,
            values: { obj: objType, objName: this.props.obj.name }
          }}
        />
      </span>
    )

  // ...

            <label for='check1'>
              <FormattedHTMLMessage {... { ...MESSAGES.submissions }} />
            </label>

  // ...

          <label for='check3'>
            <FormattedHTMLMessage {... { ...MESSAGES.entities, values: { obj: objType } }} />
          </label>

  // ...
  }
}
export default injectIntl(DeleteModal)
```